docs/languages/r.md

```diff
@@ -56,7 +56,7 @@ The R extension supports code completion and many other code editing features th
 
 ## Linting
 
-Linting is a feature that checks the code for warnings and potential errors. R code linting is provided by [lintr](https://github.com/r-lib/lintr) package. You can customize it by choosing from the list of [available linters](https://github.com/r-lib/lintr#available-linters) via the [configuration file](https://github.com/r-lib/lintr#project-configuration).
```

Comment:
Do you think the first lintr links should stay on GitHub or also move to lintr.r-lib.org?

Reply:
I think it's fine on github, lintr was recently migrated from Jim Hester's account to the r-lib organisation. That hyperlink is up to date with-respect-to the repo ownership changes. Also the github page has a link to the lintr.r-lib.org documentation.